packages/base/index.d.ts

```diff
@@ -80,6 +80,8 @@ export interface QueryOptions {
   type?: Script | "empty";
   data?: string;
   argsLen?: number;
+  fromBlock?: number;
+  toBlock?: number;
```

Comment:
Add a typedoc comment denoting this is inclusive. I know you have included a comment in the README part, but here we were generating API documents powered by typedoc, it's worth mentioning it here.

Reply:
Updated.